gwnr/waveform/esigma_utils.py

```diff
@@ -281,17 +308,29 @@ def get_inspiral_esigma_modes(
         orbital_var_dict = {}
         if return_orbital_params == True:
             return_orbital_params = orbital_var_names
+
+        if return_pycbc_timeseries:
+            for name in return_orbital_params:
+                exec(
```

Comment:
why use `exec(...)` here?

Reply:
Since I have chosen the names in `orbital_var_names` to be the same as the orbital variable names in the code, I thought this to be a straightforward way of populating the orbital variable dict. 

Let me know if I should do it in a more "explicitly-visible-in-code" way...